Domain/Sia.Core/Data/IAssociation.cs

```diff
@@ -47,6 +47,6 @@ protected BidrectionalAssociation()
 
 
         KeyValuePair<TRight, TLeft> IAssociation<TRight, TLeft>.AssociatesBetween()
-            => new KeyValuePair<TRight, TLeft>(_right, _left);
+            => new KeyValuePair<TRight, TLeft>(right, left);
     }
 }
```

Comment:
We should look at adding this into the build because coding environments can be different depending on the dev's windows git config, visual studio (or other editor) config, and possibly other places as well.